### PR TITLE
Hongtao - Full mutations coverage for ProfitsTable.js and CommonsCard.js

### LINKED ISSUE #65: Full mutation coverage for ProfitsTable.js and CommonsCard.js
Find stryker exceptions that you can remove by writing effective tests.

### PR DESCRIPTION
**Pull request by azhe12345**
## Overview

In this PR, I find stryker exceptions and remove it by writing effective tests.

## Screenshots (Optional)
Before tests:
621732614997_ pic
After tests:
631732620374_ pic
651732642326_ pic

## Validation (Optional)
1. Download this branch.
2. run npm test
3. run npx stryker run -m src/main/components/Commons/ProfitsTable.js
4. run npx stryker run -m src/main/components/Commons/CommonsCard.js
## Tests
- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #65

### CODE REVIEW

File: frontend/src/tests/components/Commons/ProfitsTable.test.js [hunk 1]
```diff
@@ -25,4 +27,32 @@ describe("ProfitsTable tests", () => {
...
+  headers.forEach((header) => {
+    expect(screen.getByText(header)).toBeInTheDocument();
+  });
+
+  const firstProfit = profitsFixtures.threeProfits[0];
+  expect(screen.getByTestId(`${testId}-cell-row-0-col-Profit`)).toHaveTextContent(
+    `$${firstProfit.amount.toFixed(2)}`
+  );
+
+  expect(screen.getByTestId(`${testId}-cell-row-0-col-date`)).toBeInTheDocument(); // Date is empty as per the current rendering
```
Comment (alecsong222) [inline frontend/src/tests/components/Commons/ProfitsTable.test.js]: Remove this code if it doesn't do anything for now

**Review by alecsong222 — COMMENTED** [review]
Remove your changes to CoursesController and CoursesControllerTests from this PR - they are not part of this issue and should not be included

File: frontend/src/tests/components/Commons/ProfitsTable.test.js (hunk 1 above)
Comment (azhe12345) [reply frontend/src/tests/components/Commons/ProfitsTable.test.js]: I think it check the date field, the comment is just when I figure the code out and forget to delete it. I will delete it and commit again

Comment (alecsong222) [reply frontend/src/tests/components/Commons/ProfitsTable.test.js]: Please update the branch to main as well

**Comment by azhe12345** [comment]
> Remove your changes to CoursesController and CoursesControllerTests from this PR - they are not part of this issue and should not be included

fix

File: frontend/src/tests/components/Commons/ProfitsTable.test.js (hunk 1 above)
Comment (azhe12345) [reply frontend/src/tests/components/Commons/ProfitsTable.test.js]: already update to main

**Pull request closed without merging**